bin/KG/dictionariesManager..js

```diff
@@ -47,6 +47,8 @@ var DictionariesManager = {
     },
 
     indexSource: function (indexName, data, _options, callback) {
+        if (typeof data === "string") data = JSON.parse(data);
+        if (typeof _options === "string") _options = JSON.parse(_options);
```

Comment:
_options was prefixed with an underscore to make eslint happy when _options was not used. Now that _options is used, let's name it options without a _ prefix.